scriptmodules/supplementary/skyscraper.sh

```diff
@@ -469,89 +419,85 @@ function gui_skyscraper() {
+        local cmd=(dialog --backtitle "$__backtitle"  --colors --cancel-label "Exit" --help-button --no-collapse --cr-wrap --default-item "$default" --menu "   Skyscraper: game scraper by Lars Muldjord ($ver)\\n \\n" 22 60 13)
 
         local -a options=(
-            1 "Scrape all systems"
-            2 "Scrape chosen systems"
+            "-" "GATHER and cache resources"
         )
 
         local source_found=0
         local online="Online"
         local i
 
+        options+=(
+            1 "Gather for All systems"
+            2 "Gather for Chosen systems"
+        )
+
         for i in "${!s_source[@]}"; do
             if [[ "$scrape_source" == "${s_source[$i]}" ]]; then
                 [[ $i -ge 10 ]] && online="Local"
-                options+=(3 "Scrape from ${s_source_names[$i]} ($online)")
+                options+=(3 "Gather source - ${s_source_names[$i]} ($online) -->")
                 source_found=1
             fi
         done
 
-        if [[ ! $source_found ]]; then
-            options+=(3 "Scrape from Screenscraper (Default)")
+        if [[ $source_found -ne 1 ]]; then
+            options+=(3 "Gather from - Screenscraper (Online) -->")
             scrape_source="screenscraper" # default scraping source if none found
         fi
 
-        if [[ "$rom_name" -eq 0 ]]; then
-            options+=(4 "ROM Names (Source name)")
-        else
-            options+=(4 "ROM Names (Filename)")
-        fi
+        options+=(4 "Cache options and commands -->")
 
-        if [[ "$remove_brackets" -eq 1 ]]; then
-            options+=(5 "Remove bracket info (Enabled)")
-        else
-            options+=(5 "Remove bracket info (Disabled)")
-        fi
+        options+=("-" "GAME LIST generation")
+        options+=(5 "Generate game list(s)")
+        options+=(6 "Generate options -->")
 
-        if [[ "$force_refresh" -eq 0 ]]; then
-            options+=(6 "Force cache refresh (Disabled)")
-        else
-            options+=(6 "Force cache refresh (Enabled)")
-        fi
+        options+=("-" "OTHER options")
 
         if [[ "$download_videos" -eq 1 ]]; then
-            options+=(7 "Download videos (Enabled)")
-        else
-            options+=(7 "Download videos (Disabled)")
-        fi
-
-        if [[ "$use_rom_folder" -eq 1 ]]; then
-            options+=(8 "Use ROM folders for gamelist & media (Enabled)")
+            options+=(V "Download videos (Enabled)")
         else
-            options+=(8 "Use ROM folders for gamelist & media (Disabled)")
+            options+=(V "Download videos (Disabled)")
         fi
 
         options+=(A "Advanced options -->")
 
+        # Show different options, depending on the previous check action
+        if [[ "$latest_ver" > "$ver" && -n "$latest_ver" ]]; then
```

Comment:
That's not a safe comparison for versions

eg (2.10 is a higher version than 2.2)
```
 [[ 2.2 > 2.10 ]] && echo "true"
```


use compareVersion from helpers.